colossus-metrics/src/main/scala/colossus/metrics/collectors/Histogram.scala

```diff
@@ -43,7 +45,49 @@ object Histogram {
     }.toVector
     BucketList(buckets)
   }
-  
+
+  /**
+    * Create a Rate with the following address.  This will use the "colossus.metrics" config path to locate configuration.
+    * @param address The MetricAddress of this Histogram.  Note, this will be relative to the containing MetricSystem's metricAddress.
+    * @param collection The collection which will contain this Collector.
+    * @return
+    */
+  def apply(address : MetricAddress)(implicit collection : Collection) : Histogram = {
+    apply(address, MetricSystem.ConfigRoot)
+  }
+
+  /**
+    * Create a Histogram with following address.  Note, the address will be prefixed with the MetricSystem's root.
+    * Configuration is resolved and overlayed as follows('metricSystemConfigPath' is the configPath parameter, if any, that was
+    * passed into the MetricSystem.apply function):
```

Comment:
I think it would make more sense to document how configuration is loaded all in one place, possibly just on `MetricSystem` or even an empty package object, since right now it seems to be scattered a bit all over and there's a lot of duplication.